test/Driver/batch_mode_print_jobs.swift

```diff
@@ -0,0 +1,13 @@
+// RUN: %empty-directory(%t)
+// RUN: touch %t/file-01.swift %t/file-02.swift %t/file-03.swift
+// RUN: echo 'public func main() {}' >%t/main.swift
+//
+// RUN: %swiftc_driver -enable-batch-mode -c -emit-module -module-name main -j 2 %t/file-01.swift %t/file-02.swift %t/file-03.swift %t/main.swift  -driver-print-jobs | %FileCheck %s -check-prefix=CHECK-COMBINED
+// RUN: %swiftc_driver -enable-batch-mode -c -emit-module -module-name main -j 2 %t/file-01.swift %t/file-02.swift %t/file-03.swift %t/main.swift  -###  | %FileCheck %s  -check-prefix=CHECK-COMBINED
+// RUN: %swiftc_driver -enable-batch-mode -driver-print-jobs -c -emit-module -module-name main -j 2 %t/file-01.swift %t/file-02.swift %t/file-03.swift %t/main.swift 2>%t/shouldBeEmpty >/dev/null && test -z `cat %t/shouldBeEmpty`
+// RUN: %swiftc_driver -enable-batch-mode -c -emit-module -module-name main -j 2 %t/file-01.swift %t/file-02.swift %t/file-03.swift %t/main.swift  -### 2>%t/shouldBeEmpty > /dev/null && test -z `cat %t/shouldBeEmpty`
+//
+//
+// CHECK-COMBINED: -primary-file {{.*}}/file-01.swift -primary-file {{.*}}/file-02.swift {{.*}}/file-03.swift {{.*}}/main.swift
+//
+// CHECK-ANYTHING-NOT: {{.+}}
```

Comment:
Looks like you're not using this anymore.

Reply:
Thanks for the catch. Removed.